### ISSUE TITLE
If an engine fails while running, it still seems to produce electric charge from it's alternator

### ISSUE THREAD

**Issue by rsparkyc** [Kerbalism/Kerbalism#747]
I had an engine fail mid-flight, but noticed that my batteries were still charging, and the alternator on the engine showed it had output. Going back to the space center and then back to the craft, the alternator was back to producing 0

**Comment by rsparkyc** [comment]
looks like this has been confirmed here: https://discord.com/channels/418222516386004992/418222516386004994/846112634196852796
and here: 
https://discord.com/channels/418222516386004992/566637552614768640/807619617424932864

**Comment by gotmachine** [comment]
Yep that's a long standing issue, quite minor. Will fix if I get some time.
